Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml

```diff
@@ -3052,6 +3038,7 @@ script:
       type: String
 
   isfetch: true
+  runonce: false
```

Comment:
Overlooked this line in previous review, please revert

Reply:
it's may added by the server 
it should be here